pyfr/plugins/tavg.py

```diff
@@ -32,8 +39,10 @@ def __init__(self, intg, cfgsect, suffix=None):
         fpdtype = self.cfg.get(cfgsect, 'precision', 'single')
         if fpdtype == 'single':
             self.fpdtype = np.float32
+            self.eps = np.finfo(np.float32).eps
```

Comment:
Given all of the accumulation and evaluation is done double precision this might not be needed.